app/metrics.yaml

```diff
@@ -1562,6 +1562,21 @@ collections:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+  total_collections:
+    type: quantity
+    unit: unit
+    gecko_datapoint: NA
```

Comment:
This is the reason why the build is not failing: you're providing a bogus `gecko_datapoint` but we have no way to tell that this is not correct :)

Reply:
Thanks for clearing up the confusion! 😄